props/prop_base.h

```diff
@@ -443,7 +443,7 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     chdir(current_preset_.font.get());
     if (on) On();
     if (announce) {
-      STDOUT << "DISPLAY: " << current_preset_name() << "\n";
+      PVLOG_STATUS << "DISPLAY: " << current_preset_name() << "\n";
```

Comment:
Change "DISPLAY: " to "Current Preset: "

Reply:
done